packages/local_auth/local_auth_darwin/darwin/Tests/FLALocalAuthPluginTests.swift

```diff
@@ -456,14 +456,34 @@ class FLALocalAuthPluginTests: XCTestCase {
       alertFactory: alertFactory, viewProvider: viewProvider)
 
     stubAuthContext.expectBiometrics = true
-    stubAuthContext.canEvaluateError = NSError(domain: "error", code: 0)
+    stubAuthContext.canEvaluateError = NSError(
+      domain: "error", code: LAError.biometryNotAvailable.rawValue)
 
     var error: FlutterError?
     let result = plugin.deviceCanSupportBiometricsWithError(&error)
     XCTAssertFalse(result!.boolValue)
     XCTAssertNil(error)
   }
 
+  func testDeviceSupportsBiometrics_withBiometryNotAvailableLoadedBiometryType() {
```

Comment:
its not super clear to me what "loadedBiometryType" means in this context
maybe testDeviceSupportsBiometrics_withBiometryNotAvailableWhenPermissionsDenied?

Reply:
Sorry, I missed this comment somehow.
https://github.com/flutter/packages/pull/8499